scripts/find_mcs_mers.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import json
+import multiprocessing as mp
+import numpy as np
+import os
+import pickle as pkl
+import sys
+
+sys.path.append(f"{os.path.dirname(os.path.abspath(__file__))}/../")
```

Comment:
We shouldn't need to do this, what are you trying to accomplish with this?

Reply:
Issue #48 is raised and we'll be working on solving this using entry points in the future. Once this PR is merged into main.